tests/Aspire.Hosting.Tests/RabbitMQ/AddRabbitMQTests.cs

```diff
@@ -24,18 +24,27 @@ public void AddRabbitMQContainerWithDefaultsAddsAnnotationMetadata()
         var containerResource = Assert.Single(appModel.Resources.OfType<RabbitMQServerResource>());
         Assert.Equal("rabbit", containerResource.Name);
 
-        var endpoint = Assert.Single(containerResource.Annotations.OfType<EndpointAnnotation>());
-        Assert.Equal(5672, endpoint.ContainerPort);
-        Assert.False(endpoint.IsExternal);
-        Assert.Equal("tcp", endpoint.Name);
-        Assert.Null(endpoint.Port);
-        Assert.Equal(ProtocolType.Tcp, endpoint.Protocol);
-        Assert.Equal("tcp", endpoint.Transport);
-        Assert.Equal("tcp", endpoint.UriScheme);
+        var primaryEndpoint = Assert.Single(containerResource.Annotations.OfType<EndpointAnnotation>().Where(e => e.Name == "tcp"));
```

Comment:
Should we make this test be a Theory and run it for both "RunMode" and "PublishMode"?